_data/simple-icons.json

```diff
@@ -1546,7 +1547,7 @@
         {
             "title": "Chevrolet",
             "hex": "CD9834",
-            "source": "https://www.chevrolet.com/content/dam/chevrolet/na/us/english/index/shopping-tools/download-catalog/02-pdf/2019-chevrolet-corvette-catalog.pdf"
+            "source": "https://media.chevrolet.com/media/us/en/chevrolet/photos.detail.html/content/Pages/galleries/us/en/vehicles/chevrolet/logos.html"
```

Comment:
I'm not sure for this one since logos from this page are in raster formats.

Reply:
I dont know. Might need to search a little bit more if suitable logos are available from an official source. The monochrome SVG in the PDF file was surely better than those raster file and it is more representative of the icon added.